### ISSUE TITLE
pod/rabbitmq-0 remains in Pending state.

### ISSUE THREAD

**Issue by MarkLFT** [bitnami/charts#4827]
**Which chart**:
bitnami/rabbitmq latest version (download fresh today,

**Describe the bug**
I have followed the instructions https://docs.bitnami.com/kubernetes/infrastructure/rabbitmq/get-started/add-repo/, and all seemed to work fine until I get the deployments, pods, services.  The services seem fine, but the pod is in a pending state, and has been for more than an hour.

I tried to remove the installation, and followed the instructions again, but received the same result. Status Pending.

**To Reproduce**

Follow the instructions in https://docs.bitnami.com/kubernetes/infrastructure/rabbitmq/get-started/add-repo/ exactly as they are types with the exception of changing the deployment name from MY-Release to rabbitmq.  That is it, that is all I did.

**Expected behavior**
I expected the rabbitmq deployment to complete normally.

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.4.2", GitCommit:"23dd3af5e19a02d4f4baa5b2f242645a1a3af629", GitTreeState:"clean", GoVersion:"go1.14.13"}
kubectl 
```

- Output of `kubectl version`:

Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.1", GitCommit:"c4d752765b3bbac2237bf87cf0b1c2e307844666", GitTreeState:"clean", BuildDate:"2020-12-18T12:09:25Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.1", GitCommit:"c4d752765b3bbac2237bf87cf0b1c2e307844666", GitTreeState:"clean", BuildDate:"2020-12-18T12:00:47Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"linux/amd64"}

**Additional context**
I have three Ubuntu 20.04.1 serves one master and two worker nodes.
This is the first helm installation I have tried.
Kuberetes works fine when I tested it with a simple nginx deployment

**Comment by marcosbc** [comment]
Hi @MarkLFT, there could be many reasons for that, i.e. not having enough resources in your cluster.

I'd recommend you to run `kubectl describe pod PODNAME` and `kubectl describe statefulset STATEFULSETNAME`, as it may show more information as to why they are stuck in PENDING state.

**Comment by maxisam** [comment]
i am having this issue too. 

It shows create Pod rabbitmq-0 in StatefulSet rabbitmq successful
 Normal  Started    6m38s  kubelet, machinename  Started container rabbitmq

But it just doesn't get ready

**Comment by juan131** [comment]
Hi @maxisam 

Could you please share the whole output of the `kubectl describe pod POD_NAME` command?
Could you also please share the output of the command `kubectl logs POD_NAME -c rabbitmq` command?

> Remember to replace the placeholder POD_NAME with the actual name of your RabbitMQ pod

**Comment by maxisam** [comment]
I figured out the issue. It was the network issue. Thx

**Comment by juan131** [comment]
Great!

**Comment by tthug** [comment]
> I figured out the issue. It was the network issue. Thx

@maxisam  Can you describe the network issue and your solution do it?

**Comment by maxisam** [comment]
@tthug  I don't remember it clearly but I think it is related to how helm chart pointing to each component. Take a deep look in the statefulset about the network / service environment variables. You can see the order doesn't make sense. 
I think the other rabbitmq chart from bitnami doesn't have this issue any more.

**Comment by tthug** [comment]
@maxisam All right thank you.

My problem was that I was deploying the chart to a EKS cluster where dynamic volume provisioning was not configured by default. This is the error I was seeing when I described the `data-rabbitmq-ha-0` pvc resource:
>  Normal  ExternalProvisioning  2m47s (x11643 over 2d)  persistentvolume-controller  waiting for a volume to be created, either by external provisioner "ebs.csi.aws.com" or manually created by system administrator

I was able to get the rabbitmq-ha pod running after installing the [Amazon EBS CSI driver](https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html) and reinstalling the chart.
